physical/etcd/etcd3.go

```diff
@@ -128,6 +122,18 @@ func newEtcd3Backend(conf map[string]string, logger log.Logger) (physical.Backen
 		return nil, err
 	}
 
+	sReqTimeout := conf["request_timeout"]
+	if sReqTimeout == "" {
+		// etcd3 default request timeout is set to 5s. It should be long enough
+		// for most cases, even with internal retry.
+		sReqTimeout = "5"
+	}
+	iReqTimeout, err := strconv.Atoi(sReqTimeout)
```

Comment:
Could you use the [ParseDurationSecond](https://github.com/hashicorp/vault/blob/143412d58e706bbf53f2f0b53fcb48966701c84b/helper/parseutil/parseutil.go#L17) function here (and bellow) instead? That will allow users to specify `15s`, which is a little more readable.

Reply:
Fixed.